### PR TITLE
CSIRO Causal Network Def and Dataset

### PR DESCRIPTION
**Pull request by nicholascar**

### CODE REVIEW

**Comment by rwalkerands** [comment]
@nicholascar : Hold this over to next control body training, or do it now?

File: conf/linked.data.gov.au/org/csiro.conf [hunk 1]
```diff
@@ -138,27 +138,27 @@ RewriteRule ^/def/project(.*)$                          https://dr-shorthair.git
...
 # https://linked.data.gov.au/dataset/causalnetwork
 # identifier maps
 # this covers /id/{name}, /id/node/{UUID}, and /id/link/{UUID}
 RewriteCond %{QUERY_STRING} ^_mediatype=text/turtle$ [OR]
 RewriteCond %{QUERY_STRING} ^_format=text/turtle$ [OR]
 RewriteCond %{HTTP:Accept} text/turtle [NC]
-RewriteRule ^/dataset/causalnetwork/id/(.*)$            https://causalnetworks.azurewebsites.net/causalnetwork/id/(.*).ttl [R=302,QSA,L]
-RewriteRule ^/dataset/causalnetwork/id/(.*)$            https://causalnetworks.azurewebsites.net/causalnetwork/id/(.*) [R=302,QSA,L]
-RewriteRule ^/dataset/causalnetwork/(.*)$               https://causalnetworks.azurewebsites.net/causalnetwork/(.*) [R=302,QSA,L]
+RewriteRule ^/dataset/causalnetwork/id/(.*)$            https://causalnetworks.azurewebsites.net/causalnetwork/id/$1.ttl [R=302,QSA,L]
```
Comment (ashleysommer) [inline conf/linked.data.gov.au/org/csiro.conf:162]: Thanks for finding this regex substitution bug I missed!

**Comment by rwalkerands** [comment]
> This has been approved by the Control Body so now that tests pass, it can be merged

True, but not an answer to my question.

I'll answer it myself: "Do it now."

**Review by rwalkerands — APPROVED** [review]

**Comment by rwalkerands** [comment]
Deployed.

**Pull request merged**